src/secp256k1/include/secp256k1_schnorrsig.h

```diff
@@ -0,0 +1,125 @@
+#ifndef SECP256K1_SCHNORRSIG_H
+#define SECP256K1_SCHNORRSIG_H
+
+#include "secp256k1.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** This module implements a variant of Schnorr signatures compliant with
+ * BIP-schnorr
```

Comment:
```suggestion
 * BIP-340
```

Reply:
This belongs in https://github.com/bitcoin-core/secp256k1/pull/558.